CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Change Log
 ==========
 
+0.8.2
+-----
+
+* `DjangoApp`: Correctly detect Django version used. Until now yodeploy
+  has incorrectly assumed >= 1.7, when performing migrations.
```

Comment:
I can always git blame to find the change. esp if the changelog entry was in the PR that made the change.